src/helpers.ts

```diff
@@ -227,4 +227,4 @@ function noopWarn(..._: any[]): void {
 }
 
 export const warn: (...args: any[]) => void =
-  process?.env?.JEST_WORKER_ID === undefined ? consoleWarn : noopWarn;
+  (typeof process === 'undefined' || process?.env?.JEST_WORKER_ID === undefined) ? consoleWarn : noopWarn;
```

Comment:
yup, this makes sense.  just change the single-quotes to backticks to make my lint rule happy, and I'll merge.  thanks!

```suggestion
  (typeof process === `undefined` || process?.env?.JEST_WORKER_ID === undefined) ? consoleWarn : noopWarn;
```